core/src/pages.rs

```diff
@@ -56,7 +56,7 @@ const _: () = assert!(WASM_PAGE_SIZE % GEAR_PAGE_SIZE == 0);
     Default,
     derive_more::Into,
 )]
-pub struct PagesAmount<const SIZE: u32>(u32);
+pub struct PagesAmount<const SIZE: u32>(pub(crate) u32);
```

Comment:
It's not safe to do this, because `gear-core` is big enough, so developers can simply make bugs using this constructor. `PagesAmount` supports `Into<u32>`, `From<u16>` and `TryFrom<u32>` this must be enough for any purposes in tests.

Reply:
Or you can make it `pub` for tests only